packages/data/addon/utils/metric.js

```diff
@@ -122,3 +122,22 @@ export function parseMetricName(canonicalName) {
     parameters
   };
 }
+
+/**
+ * Returns a metric object given column attributes
+ * @function mapColumnAttributes
+ * @param {Object} attributes - column attributes
+ * @param {String} attributes.name - metric name
+ * @param {Object} attributes.parameters - metric parameters
+ * @returns {Object} - metric object with metric (name) and parameters
```

Comment:
Should we just say `object with metric name and params` here? metric object made me think of the metric metadata object or the fragment.